src/invoices/InvoiceForm/CurrentExchangeRate/CurrentExchangeRate.js

```diff
@@ -0,0 +1,68 @@
+import React, {
+  useState,
+  useEffect,
+} from 'react';
+import PropTypes from 'prop-types';
+
+import { stripesConnect } from '@folio/stripes/core';
+import {
+  KeyValue,
+  Label,
+  Loading,
+} from '@folio/stripes/components';
+
+import { EXCHANGE_RATE_API } from '../../../common/constants/api';
+
+const CurrentExchangeRate = ({ label, exchangeFrom, exchangeTo, mutator }) => {
+  const [exchangeRate, setExchangeRate] = useState();
+
+  useEffect(
+    () => {
+      setExchangeRate();
+
+      mutator.exchangeRate.GET({
+        path: `${EXCHANGE_RATE_API}?from=${exchangeFrom}&to=${exchangeTo}`,
```

Comment:
imho no need doing this, you can separate `path` and `params`

Reply:
Also path is static and could be set in manifest definition

Reply:
But if I try to add `params: { query:  from=${exchangeFrom}&to=${exchangeTo} }` the response returns with error because `query=` is not needed in the URL `https://folio-snapshot-okapi.dev.folio.org/finance/exchange-rate?query=from%3DEUR%26to%3DUSD`

Reply:
so why do you trying to add `query` if it doesn't work and returns an error?

Reply:
According to [Stripes API Connect](https://github.com/folio-org/stripes-connect/blob/master/doc/api.md#rest-resources) `params` it is an object with named parameters

Reply:
still it's not clear why do you want to use `query`

Reply:
I mean `params: from=${exchangeFrom}&to=${exchangeTo}` would not work

Reply:
???
```
mutator.exchangeRate.GET({
  params: {
    from: exchangeFrom,
    to: exchangeTo,
  },
});
```

Reply:
Thanks 🙏